compose/ccd-next-hearing.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Don't forget changes to the `ccd` script - to be able to use the commands like `./ccd set <app> <branch>` - and the README.
See https://github.com/hmcts/ccd-docker/pull/162 for a previous example for case disposer.

Reply:
DONE :)